### PR TITLE
login page frontend

### PR DESCRIPTION
**Pull request by jhoannaricaa**
## Task Link

- https://app.asana.com/0/1202718495875213/1202725278795534/f

## Commands

- cd frontend
- npm start

## Screenshots
image

### CODE REVIEW

File: frontend/src/components/Template/NavBar.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 .signup-btn {
   position: absolute;
-  right: 10px;
+  right: 10%;
```
Comment (chevyB) [inline frontend/src/components/Template/NavBar.css:7]: Your buttons are not consistent. Your signup btn is 10% while login is 10px
Also avoid using absolute as much as possible if you can make it work without it.
You may want to use flex box instead.

File: frontend/src/components/Template/NavBar.css [hunk 2]
```diff
@@ -59,3 +59,19 @@
...
+  width: 8%;
+  height: 32px;
+  border-radius: 10px;
+  padding-bottom: 5%;
+  outline: none;
+  border: none;
+  color: #fff;
+  font-size: 1rem;
+  text-align: start;
+}
```
Comment (chevyB) [inline frontend/src/components/Template/NavBar.css:77]: add new line at EOF

File: frontend/src/components/Template/NavBar.js [hunk 3]
```diff
@@ -57,11 +62,45 @@ const NavBar = () => {
...
           }
         />
       </div>
+    
     </>
   );
 };
 
-export default NavBar;
+export default NavBar;
```
Comment (chevyB) [inline frontend/src/components/Template/NavBar.js:106]: add new line at EOF

File: frontend/src/pages/LogInPage/LogInPage.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    justify-content: left;
+    align-items: left;
+    width: 495px;
+    height: 320px;
+
+}
+
+.password{
+    margin-top: 10px;
+
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.css:44]: remove white space

File: frontend/src/pages/LogInPage/LogInPage.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+.auth-button{
+    margin-top: 10px;
+    margin-right: 10px;
+}
+
+.email{
+    margin-top: 10px;
+    display:flex;
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.css]:
```suggestion
.email {
    margin-top: 10px;
    display: flex;
```
add spaces, also apply to others

File: frontend/src/pages/LogInPage/LogInPage.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+.placholders-temp{
+
+    margin-left: 100px;
+    margin-right: 100px;
+}
+
+.post-matters{
+    margin-top: 20px;
+}
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.css]: add new line at EOF

File: frontend/src/pages/LogInPage/LogInPage.css [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+.avatar{
+    width: 50px;
+    height: 50px;
+    border-radius: 100%;
+    border: 4px solid gray;
+    margin-right: 20px;
+}
+
+.placholders-temp{
+
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.css:115]: remove white space

File: frontend/src/pages/LogInPage/LogInPage.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './LogInPage.css';
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.js:2]: Note that this type of import will be applied globally (whole app).
 So make sure your classname is unique or you may want to use modular css next time

File: frontend/src/pages/LogInPage/LogInPage.js [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+              />
+              {/* {errors.password && <p>{errors.password}</p>} */}
+            </div>
+           
+            <button>
+              Show Password
+            </button>
+            <div>
+            <div className='ui hidden divider'></div>
+            <div className='ui hidden divider'></div>
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.js]: If there are more than 1 divider, just use padding or margin

File: frontend/src/pages/LogInPage/LogInPage.js [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+            <button className="sign-up-form-input-btn" type="submit">
+            Login
+            </button>
+            </div>
+            <div className='ui hidden divider'></div>
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.js]:
```suggestion
            <div className='ui hidden divider' />
```
You can use single tag

File: frontend/src/pages/LogInPage/LogInPage.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+            <span className="sign-up-form-input-login">
+              Forgot Password? 
+            </span>
+          </div>
+        </div>
+      );
+}
+
+export default Login
```
Comment (chevyB) [inline frontend/src/pages/LogInPage/LogInPage.js]: add new line at EOF

**Review by chevyB — APPROVED** [review]

**Review by chevyB — APPROVED** [review]

File: frontend/src/pages/LogInPage/LogInPage.js [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+              />
+              {/* {errors.email && <p>{errors.email}</p>} */}
+            </div>
+            <div className="login-input">
+              <input
+                className="login-text-input"
+                type="password"
+                name="password"
+                placeholder="Password"
+                value="Password"
```
Comment (velezrenz) [inline frontend/src/pages/LogInPage/LogInPage.js:28]: Please remove 
value = "Password"

**Pull request merged**